pydantic/errors.py

```diff
@@ -204,3 +204,10 @@ class UUIDVersionError(PydanticValueError):
 
     def __init__(self, *, required_version: int) -> None:
         super().__init__(required_version=required_version)
+
+
+class ArbitraryTypeError(PydanticTypeError):
```

Comment:
I think for better readability we need to define `code = 'type_error.arbitrary_type'` or rename class to `ArbitraryError`